component/componentprofiles/config.go

```diff
@@ -0,0 +1,15 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
I suppose you will add the interface in a separate PR?

Reply:
We discussed this privately. This was a red herring, as the component package has no interface to implement.